### DISCUSSION TITLE
Weird display of web interface

### DISCUSSION THREAD

**Discussion by Guddi11** [MiczFlor/RPi-Jukebox-RFID#2059] [Q&A] [answered]
Hi, since the last install of my phoniebox (2.4) whenever I access the web interface, it displays parts of the phoniebox configuration at the top. Does anybody know where this behaviour might come from (see pictures below)?

Apart from that, everything seems to work fine, so it is rather a beauty issue... . Thanks in advance.

![image](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/22029082/c2039d25-7d18-46dd-863e-339da507873a)
![image](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/22029082/0c190a84-4226-4f87-8a0b-dfaa9d1e7ee6)

**Comment by s-martin** [comment, marked as answer]
Please check here https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/MANUAL#debug-log-settings
